src/inject/dynamic-theme/stylesheet-modifier.ts

```diff
@@ -138,7 +138,11 @@ export function createStyleSheetModifier() {
                 return `${property}: ${value == null ? sourceValue : value}${important ? ' !important' : ''};`;
             };
 
-            const ruleText = `${selector} { ${declarations.map(getDeclarationText).join(' ')} }`;
+            let cssRulesText = '';
+            declarations.forEach((declarations) => {
```

Comment:
Wonderful! One note: `declarations` variable can cause some confusion in understanding, because the same name is used for another variable on the same line.